Assets/MRTK.ThirdParty/MRTK-Quest/Editor/MRTKQuestConfigurationChecker.cs

```diff
@@ -0,0 +1,203 @@
+                        {
+                            // Add the new warning numbers if they are not already in the file
+                            if (!warningNumbers.Contains(warningNumberToAdd))
+                            {
+                                warningNumbers.Add(warningNumberToAdd);
+                            }
+                        }
+
+                        cscFileLines.Add(string.Join(",", warningNumbers));
+                    }
+                    else
+                    {
+                        cscFileLines.Add(cscFileLine);
+                    }
+                }
+            }
+
+            using (StreamWriter streamWriter = new StreamWriter(cscFilePath))
+            {
+                foreach (string cscLine in cscFileLines)
+                {
+                    if (cscLine.StartsWith("1701"))
+                    {
+                        string warningNumbersJoined = string.Join(",", warningNumbers);
+                        streamWriter.WriteLine(string.Concat("-nowarn:", warningNumbersJoined));
+                    }
+                    else
+                    {
+                        streamWriter.WriteLine(cscLine);
+                    }
+                }
+            }
+
+            //Debug.Log($"Saving {cscFilePath}");
+        }
+
+
+        /// <summary>
+        /// Locates the files that match the specified name within the Assets folder structure.
+        /// </summary>
+        /// <param name="fileName">The name of the file to locate (ex: "TestFile.asmdef")</param>
+        /// <returns>Array of FileInfo objects representing the located file(s).</returns>
+        public static FileInfo[] FindFilesInAssets(string fileName)
+        {
+            // FindAssets doesn't take a file extension
+            string[] assetGuids = AssetDatabase.FindAssets(Path.GetFileNameWithoutExtension(fileName));
+
+            List<FileInfo> fileInfos = new List<FileInfo>();
+            for (int i = 0; i < assetGuids.Length; i++)
+            {
+                string assetPath = AssetDatabase.GUIDToAssetPath(assetGuids[i]);
+                // Since this is an asset search without extension, some filenames may contain parts of other filenames.
+                // Therefore, double check that the path actually contains the filename with extension.
+                if (assetPath.Contains(fileName))
+                {
+                    fileInfos.Add(new FileInfo(assetPath));
+                }
+            }
+
+            return fileInfos.ToArray();
+        }
+
+        /// <summary>
+        /// Appends a set of symbolic constant definitions to Unity's Scripting Define Symbols for the
+        /// specified build target group.
+        /// </summary>
+        /// <param name="targetGroup">The build target group for which the symbols are to be defined.</param>
+        /// <param name="symbols">Array of symbols to define.</param>
+        public static void AppendScriptingDefinitions(
```

Comment:
Similar comment to the ScriptingUtilities.AppendScriptingDefinitions/RemoveScriptingDefinitions